firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -6894,6 +6894,53 @@ extension String {
     }
 }
 
+
+// MARK: - Password generator
+extension String {
+    public struct PasswordGenerator {
+        public static let UseStrongPassword = MZLocalizedString(
+            key: "PasswordGenerator.UseStrongPassword.v132",
```

Comment:
I noticed in a lot of places, for the variable name and key we often use descriptors like "Title", "Body", "Description". Do you think that style makes sense here, and for some of the other new strings?

Reply:
Yep, will do!